cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsStackValidator.java

```diff
@@ -24,15 +42,58 @@
 
     @Override
     public void validate(AuthenticatedContext ac, CloudStack cloudStack) {
+        validateStackNameAvailability(ac);
+        validateInstanceStorageOnInstanceTypes(ac, cloudStack);
+    }
+
+    private void validateStackNameAvailability(AuthenticatedContext ac) {
         AwsCredentialView credentialView = new AwsCredentialView(ac.getCloudCredential());
         String regionName = ac.getCloudContext().getLocation().getRegion().value();
         AmazonCloudFormationClient cfClient = awsClient.createCloudFormationClient(credentialView, regionName);
         String cFStackName = cfStackUtil.getCfStackName(ac);
         try {
+            LOGGER.debug("Checking stack name availability. [{}]", cFStackName);
             cfClient.describeStacks(new DescribeStacksRequest().withStackName(cFStackName));
             throw new CloudConnectorException(String.format("Stack is already exists with the given name: %s", cFStackName));
         } catch (AmazonServiceException ignored) {
+            LOGGER.warn("Exception while checking stack name availability.", ignored);
```

Comment:
can we add a check that the catched exception is notfound or not, if it is we should log only a single line on info that we are good to go with the stack name, otherwise we should log on warn level.